<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="start_pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Process Generation</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="numProcess_spinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="generationType_comboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="processType_comboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Process #</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>How Many</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Process Type</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="processNumber_spinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="pause_checkBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause?</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="numCycles_spinBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause after how many cycles?</string>
    </property>
   </widget>
   <widget class="QLabel" name="processStats_label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Processes</string>
    </property>
   </widget>
   <widget class="QLabel" name="newProcess_label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NEW: </string>
    </property>
   </widget>
   <widget class="QLabel" name="waitingProcess_label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>WAITING:</string>
    </property>
   </widget>
   <widget class="QLabel" name="terminatedProcess_label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TERMINATED:</string>
    </property>
   </widget>
   <widget class="QLabel" name="runningProcess_label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RUNNING:</string>
    </property>
   </widget>
   <widget class="QLabel" name="readyProcess_label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>READY:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="statistics_pushButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>410</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
